astro_2.0/features/Discover/components/Governance/Governance.tsx

```diff
@@ -129,15 +164,34 @@ export const Governance: FC = () => {
     }
 
     if (chart && isMounted()) {
-      setChartData(
-        chart.data.metrics.map(({ timestamp, count }) => ({
+      let newData;
+
+      if (activeView === GovernanceTabs.VOTE_THROUGH_RATE) {
+        newData = chart.data.metrics.map(({ timestamp, count }) => ({
```

Comment:
`chart.data.metrics` is being used here and in line 175. Won't it be more fancy to place it in a variable/destruct it?

Reply:
Thanks. Will be fixed in tech debt.